xdsl/backend/riscv/register_allocation.py

```diff
@@ -49,6 +49,32 @@ def allocate(self, reg: SSAValue) -> bool:
 
         return False
 
+    def process_operation(self, op: Operation) -> None:
+        match op:
+            case riscv_scf.ForOp():
+                self.allocate_for_loop(op)
+            case RISCVOp():
+                self.process_riscv_op(op)
+            case _:
+                return
```

Comment:
Would it make sense to raise an exception here?

Reply:
I added a comment, I think we just want to keep ignoring non-riscv ops for now